pkg/kubernetes/watcher.go

```diff
@@ -240,26 +241,25 @@ func (w *ObjListWatcher) handleAdd(obj interface{}) {
 			klog.V(5).Infof("Pod %s %s is ready with %d container statuses, %d init container status, %d ephemeral statues",
 				pod.Name, pod.Namespace, len(pod.Status.ContainerStatuses), len(pod.Status.InitContainerStatuses), len(pod.Status.EphemeralContainerStatuses))
 			w.Mx.Lock()
-			err1 := w.fillInfo(pod, pod.Status.ContainerStatuses)
-			err2 := w.fillInfo(pod, pod.Status.InitContainerStatuses)
-			err3 := w.fillInfo(pod, pod.Status.EphemeralContainerStatuses)
+			w.fillInfo(pod, pod.Status.ContainerStatuses)
```

Comment:
It's probably better to return an error here?
If we bubble that error back up to `handleEvent` then the item will get requeued (after backoff) so we can attempt to get the necessary ID's again.

Reply:
Also note the `continue` on line 238 above, that needs fixing too 😄

Reply:
Cool, I will fix this up. Thanks for the suggestion